### PR TITLE
templates: remove gemini_function_agent unused file

### PR DESCRIPTION
**Pull request by Sanjaypranav**
- [X] Gemini Agent Executor imported  `agent.py` has Gemini agent executor which was not utilised in current template of gemini function agent 🧑‍💻 instead openai_function_agent has been used

@sbusso  @jarib  please someone review it

### CODE REVIEW

**Comment by sbusso** [comment]
@Sanjaypranav please do not tag randomly

**Comment by Sanjaypranav** [comment]
ok kindly someone review

File: templates/gemini-functions-agent/main.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-from openai_functions_agent.agent import agent_executor
-
+from gemini_functions_agent import agent_executor # from current folder 
+# Issue Not solved
```
Comment (baskaryan) [inline templates/gemini-functions-agent/main.py:2]: what does this mean?

**Review by efriis — APPROVED** [review]
This file is unused so removing from template instead

**Pull request merged**